lib/ldp.js

```diff
@@ -383,7 +383,7 @@ LDP.prototype.get = function (host, reqPath, baseUri, includeBody, contentType,
 
     // Just return, since resource exists
     if (!includeBody) {
-      return callback(null, stats)
+      return callback(null, stats, contentType, stats.isDirectory())
```

Comment:
The problem occurs later on, since `container` will be undefined. This is the relevant line: https://github.com/linkeddata/ldnode/blob/fix-redirect/lib/handlers/get.js#L64